osler/core/api/test.py

```diff
@@ -20,16 +20,7 @@ class APITest(APITestCase):
     fixtures = [BASIC_FIXTURE]
 
     def setUp(self):
-        workupModels.ClinicType.objects.create(name="Basic Care Clinic")
-        workupModels.ClinicDate.objects.create(
-            clinic_type=workupModels.ClinicType.objects.first(),
-            clinic_date=now().date() + datetime.timedelta(days=1))
-        workupModels.ClinicDate.objects.create(
-            clinic_type=workupModels.ClinicType.objects.first(),
-            clinic_date=now().date() - datetime.timedelta(days=1))
-        workupModels.ClinicDate.objects.create(
-            clinic_type=workupModels.ClinicType.objects.first(),
-            clinic_date=now().date() - datetime.timedelta(days=5))
+        models.EncounterStatus.objects.create(name="Active",is_active=True)
```

Comment:
You should be able to put this into the core fixture, and load the fixture during tests

Reply:
yeah sorry like I said I know it's all a terrible mix of the old way and factories...